lib/git-lfs-s3/services/ceph_presigner.rb

```diff
@@ -0,0 +1,24 @@
+require 'base64'
+require 'date'
+require 'digest/sha1'
+require 'openssl'
+require 'uri'
+
+module GitLfsS3
+  module CephPresignerService
+    extend self
+    extend AwsHelpers
+
+    def signed_url(obj)
+      expire_at = (DateTime.now + 1).strftime("%s")
+      secret_access_key = Aws.config[:secret_access_key]
+      access_key_id = Aws.config[:access_key_id]
```

Comment:
I believe that this is asking the aws-sdk to resolve these values from scratch.  I don't know if it's actually possible but I could imagine this coming up with a different answer (the aws-sdk is doing a lot... not sure the exact resolution order) than was set from the env.

What about looking up the values that were set the sinatra app hash? (GitLfsS3::Application)